apps/web/src/components/dashboard/energy-aggregation-option.tsx

```diff
@@ -1,31 +1,37 @@
-"use client";
+import React, { useMemo } from 'react';
```

Comment:
Auch hier wieso wurde "use client" entfernt?
Dadurch schlägt der build in web auch fehl.